amundsen_application/static/js/fixtures/globalState.ts

```diff
@@ -29,6 +29,28 @@ const globalState: GlobalState = {
     myBookmarksIsLoaded: false,
     bookmarksForUser: [],
   },
+  dashboard: {
```

Comment:
Creating separate fixtures for chunks of data will be helpful moving forward because not doing so doesn't scale well. For example separate sub-fixtures for the default state or some initial dummy data can be imported from `fixtures/dashboards` can be used here, and in future tests across `ducks` and `components`